### PR TITLE
[Keymap] Planck Keymap: tomkonidas

### PR DESCRIPTION
**Pull request by tomkonidas**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/planck/keymaps/tomkonidas/config.h [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+   - MIDI notes can be added to the keymap
+   - Octave shift and transpose
+   - Virtual sustain, portamento, and modulation wheel
+   - etc.
+*/
+//#define MIDI_ADVANCED
+
+/* override number of MIDI tone keycodes (each octave adds 12 keycodes and allocates 12 bytes) */
+//#define MIDI_TONE_KEYCODE_OCTAVES 2
+
```
Comment (fauxpark) [inline keyboards/planck/keymaps/tomkonidas/config.h:47]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/planck/keymaps/tomkonidas/readme.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+Created by [@tomkonidas](https://tomkonidas.com) - feel free to contact me!
+
+<p>
+  <a href="https://github.com/tomkonidas">
+    <img alt="GitHub followers" src="https://img.shields.io/github/followers/tomkonidas?label=Follow&style=social">
+  </a>
+  <a href="https://twitter.com/tomkonidas">
+    <img alt="Twitter Follow" src="https://img.shields.io/twitter/follow/tomkonidas?label=Follow&style=social">
+  </a>
+</p>
```
Comment (drashna) [inline keyboards/planck/keymaps/tomkonidas/readme.md:114]: Please remove all html tags.

Comment (tomkonidas) [reply keyboards/planck/keymaps/tomkonidas/readme.md]: Removed them 👉 8919627

**Review by drashna — APPROVED** [review]

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**